lighthouse-core/audits/dobetterweb/uses-request-compression.js

```diff
@@ -48,29 +48,36 @@ class UsesRequestCompression extends Audit {
   static audit(artifacts) {
     const networkRecords = artifacts.networkRecords[Audit.DEFAULT_PASS];
 
-    // Filter requests that are on the same host as the page and not over h2.
+    // Filter requests that are text based and have gzip/br encoding.
     const resources = networkRecords.filter(record => {
       return record._resourceType && record._resourceType._isTextType &&
-        !record._responseHeaders.find(header =>
-          header.name.toLowerCase() === 'content-encoding' &&
-          (header.value === 'gzip' || header.value === 'br')
-        );
+          record._resourceSize > 1000 &&
```

Comment:
Ah that would make sense. Let's add a comment if that's the case.

If we go the threshold route, should we standardize on the same threshold as the image opt audit (and future audits). That's also 10%?